### PR TITLE
Azure Stack documentation overhaul

### PR DESCRIPTION
**Pull request by BaileyLawson**
Minor improvements made to all documents for Azure Stack.

These changes include grammatical fixes, updates, formatting fixes etc....

### CODE REVIEW

File: articles/azure/azs-gs.md [hunk 1]
```diff
@@ -103,15 +103,13 @@ After providing the above details, you should receive a welcome email shortly fr
 
 ***
 
-## Logging in to the Microsoft Azure Stack portal
+## Logging in to the UKCloud for Microsoft Azure portal
 
 To manage UKCloud for Microsoft Azure, you need to log in to the Azure Stack portal. The portal provides access to all your Azure resources.
```
Comment (suehighmoor) [inline articles/azure/azs-gs.md]: Change this to UKCloud for Microsoft Azure portal too?

File: articles/azure/azs-gs.md [hunk 2]
```diff
@@ -103,15 +103,13 @@ After providing the above details, you should receive a welcome email shortly fr
...
 
 To log in to the Azure Stack portal:
 
-1. In your browser, go to the following URL:
-
-    <https://portal.frn00006.azure.ukcloud.com/>
+1. In your browser, go to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com).
```
Comment (suehighmoor) [inline articles/azure/azs-gs.md]: Change this to UKCloud for Microsoft Azure portal too?

File: articles/azure/azs-gs.md [hunk 3]
```diff
@@ -103,15 +103,13 @@ After providing the above details, you should receive a welcome email shortly fr
 
 ***
 
-## Logging in to the Microsoft Azure Stack portal
+## Logging in to the UKCloud for Microsoft Azure portal
```
Comment (suehighmoor) [inline articles/azure/azs-gs.md]: I think elsewhere in the doc we call this the UKCloud Azure Stack portal, so maybe change to that instead? Also, if you change what we call it here, you need to change it in the rest of the article as well.

File: articles/azure/azs-how-deploy-asdk.md [hunk 4]
```diff
@@ -20,28 +20,26 @@ The Azure Stack Development Kit (ASDK) is a single server instance of Azure Stac
...
+| RAM        | \>=128GB        |
+| OS Disks   | \>=200GB        |
+| Data Disks | \>=4 \* \>=250G |
 
-**Virtualisation Note:** VMware HW must be >=11, CPU virtual extension pass-through must be enabled
+**Virtualisation Note:** VMware HW must be \>=11, CPU virtual extension pass-through must be enabled
 
-> [!IMPORTANT]
-> If you deploy your VM with Hardware Version 11 and Operating System Family as Windows Server 2016 you will not be able to create S2D Cluster as disk UUIDs are not exposed.
+> \[!IMPORTANT\] If you deploy your VM with Hardware Version 11 and Operating System Family as Windows Server 2016 you will not be able to create S2D Cluster as disk UUIDs are not exposed.
```
Comment (suehighmoor) [inline articles/azure/azs-how-deploy-asdk.md]: Why did you add the \? We don't normally need to escape these characters

File: articles/azure/azs-gs.md (hunk 3 above)
Comment (BaileyLawson) [reply articles/azure/azs-gs.md]: We call it:
- Azure Stack portal
- UKCloud for Microsoft Azure portal
- UKCloud Azure Stack portal
- Microsoft Azure Stack Portal

I 100% agree that we should keep it consistent across all of our docs, but I'm not sure which is the best option. I'm leaning towards "Azure Stack portal" or "UKCloud Azure Stack portal" though, thoughts?

File: articles/azure/azs-how-deploy-asdk.md (hunk 4 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-deploy-asdk.md]: My bad, I forgot how terrible "format document" in VSCode is for Markdown..... Fixed now

File: articles/azure/azs-gs.md (hunk 3 above)
Comment (suehighmoor) [reply articles/azure/azs-gs.md]: I'd suggest we go with UKCloud Azure Stack portal to identify it as specific to us. That's pretty cumbersome, so what we could do is use that full name for the first use in each article and then just use Azure Stack portal for the rest.

Comment (BaileyLawson) [reply articles/azure/azs-gs.md]: Have made changes to all mentions of the portal. It's a mixture between UKCloud Azure Stack portal and Azure Stack portal, using the first for headers, the first mentions of the portal in docs and for overview docs (Getting started, FAQs, et cetera.). I used the latter for everywhere else

Comment (suehighmoor) [reply articles/azure/azs-gs.md]: Good approach, thanks
